README.rst

```diff
@@ -321,7 +322,7 @@ Android Studio Workarounds
 The following factors can all contribute to some instability in the Android
 Studio managed build:
 
-* Android Studio 3.2 is **beta**
+* Android Studio 3.2.1
```

Comment:
It was an instability item, since it's not beta anymore then this line can be removed.

Reply:
i see that now.  i'll merge this and then update the readme without the ci.